apps/prairielearn/assets/scripts/instructorAssessmentManualGradingAssessmentQuestionClient.ts

```diff
@@ -431,6 +439,23 @@ onDocumentReady(() => {
             return 1;
           }
         });
+      } else if (sortName === 'ai_graded') {
+        const aiGradingStatusOrder = { Graded: 1, LatestRubric: 2, OutdatedRubric: 1, None: 0 };
+        data.sort(function (a, b) {
+          if (
+            aiGradingStatusOrder[a.ai_grading_status] < aiGradingStatusOrder[b.ai_grading_status]
+          ) {
+            return order * -1;
+          } else if (
+            aiGradingStatusOrder[a.ai_grading_status] > aiGradingStatusOrder[b.ai_grading_status]
+          ) {
+            return order;
+          } else {
+            return (
+              ((a.last_human_grader ?? '') < (b.last_human_grader ?? '') ? 1 : -1) * order * -1
```

Comment:
I think you can simplify this somewhat (I tested this in isolation in a Node REPL, but confirm that this does the right thing in the context of the overall sort):

```suggestion
              (a.last_human_grader ?? '').localeCompare(b.last_human_grader ?? '') * order
```

You can probably simplify things as a whole too:

```ts
const aOrder = aiGradingStatusOrder[a.ai_grading_status];
const bOrder = aiGradingStatusOrder[b.ai_grading_status];
if (aOrder !== bOrder) {
  return (aOrder - bOrder) * order;
}
return ((a.last_human_grader ?? '').localeCompare(b.last_human_grader ?? '')) * order;
```

(Also untested, try it out before accepting)